app/controllers/application_controller.rb

```diff
@@ -322,7 +322,9 @@ def saved_redirect_url_or(default)
 
   def generic_exception(exception)
     Foreman::Logging.exception("Action failed", exception)
```

Comment:
Should this change to actually provide the exception message and/or code instead of just "Action failed" which doesn't give any info?